kolibri/core/assets/src/views/core-modal.vue

```diff
@@ -56,40 +101,57 @@
         type: String,
         required: true,
       },
-      invisibleTitle: {
+      submitText: {
+        type: String,
+        required: false,
+      },
+      cancelText: {
+        type: String,
+        required: false,
+      },
```

Comment:
I'm a bit torn. On one hand, I appreciate that this saves a bunch of boilerplate. On the other hand, keeping it as a slot would be more flexible if (and probably when) we decide to make an exception to the rule and change the button arrangement.

My inclination is to go with the more flexible option and keep it a slot, or at least to preserve that opportunity if need-be...

Reply:
Added an actions slot.